tests/antispoof/tests.c

```diff
@@ -266,6 +266,23 @@ static void test_antispoof_generic(const unsigned char **pkts,
 	pg_antispoof_arp_del_all(antispoof); 
 	REPLAY(0);
 
+	/* add ARP_MAX */
+       for (int i=0;i<=100;i++){
+	       if (i!=100){
+		       g_assert(!pg_antispoof_arp_add(antispoof, i, &error));
+                       g_assert(!error);
+	       }else {
+		       g_assert(pg_antispoof_arp_add(antispoof, i, &error));
+	       }
+       }	       
+       g_assert(error);
```

Comment:
and this, should be in the same block, so you should either move what's inside the else block here, or move what's here in the else block